source/codegen/metadata/nidaqmx/functions_addon.py

```diff
@@ -1,2 +1,36 @@
 functions_override_metadata = {
 }
+
+functions_validation_suppressions = {
```

Comment:
The suppressions look good. There aren't as many as I feared and the syntax makes sense.